packages/core/src/services/scales-cartesian.ts

```diff
@@ -214,10 +214,14 @@ export class CartesianScales extends Service {
 
 	protected findMainVerticalAxisPosition() {
 		const options = this.model.getOptions();
-		const axisOptions = Tools.getProperty(options, "axes");
+		const axesOptions = Tools.getProperty(options, "axes");
 
 		// If right axis has been specified as `main`
-		if (Tools.getProperty(axisOptions, AxisPositions.RIGHT, "main") === true) {
+		if (
+			(Tools.getProperty(axesOptions, AxisPositions.LEFT) === null &&
```

Comment:
Indentation makes it looks like the grouping is opposite from what it is.
Also braces around the && are not actually needed.

Reply:
could you write the if-statement as you're imagining it please? I didn't really understand your comment there. Also we've got #617 which adds prettier

Reply:
I really dislike prettier and don't think we should add it.

Reply:
There are so many reasonable ways to do it, feel free to pick one that suggests the right operator priority.

Reply:
the point of prettier is to get rid of our different opinions